qa/L0_dont_release_after_unload/scripts/compare_memory_usage.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
```

Comment:
How about, instead of creating two separate tests for that, compare memory usage in a single case but twice (mem before loading, after loading and after unloading)